### ISSUE TITLE
[Bug] Visual glitch with hamsterballed hamster and enchanted items

### ISSUE THREAD

**Issue by Tschipp** [capnkirok/animaniamod#150]
_From @Deshiba on February 25, 2018 9:5_

### Expected Behavior
No visual glitches

### Actual Behavior
Visual glitch with enchanted tool when a hamster balled hamster is on the shoulder and the item is equiped.

### Steps to Reproduce
Get an enchanted tool
Get a hamster
Get your hamster in a hamster-ball
Carry the hamster on your shoulder
Equip the enchanted tool

### Version of Minecraft, version of Animania, Single Player or Server
This is on the community server.
Minecraft version: 1.12.2
Animania version: 1.4.6
Crafstudio version: 1.0.0
Decocraft version: 2.5.4
JEI version: 4.8.5.147
Hwyala version: 1.8.23

### Screenshots encouraged
![2018-02-25_10 01 30](https://user-images.githubusercontent.com/10565800/36639723-389e2bd2-1a13-11e8-99a4-06cae9019e30.png)

_Copied from original issue: capnkirok/animania#585_

**Comment by Tschipp** [comment]
_From @capnkirok on February 25, 2018 10:37_

Yeah, I need to change a few things with the Hamster Ball. One is that you shouldn't be able to put it on your shoulder at all (which would solve this problem).

**Comment by Tschipp** [comment]
_From @Deshiba on February 25, 2018 11:45_

Shift+right click should be used to have the hamster exit the ball. As you stated that would solve this issue

**Comment by Tschipp** [comment]
_From @Deshiba on March 4, 2018 10:50_

The glitch happens with just the hamster as well, no ball gives the same effect on enchanted items.

**Comment by Echoweaver** [comment]
Just spent an evening and morning figuring out that my sudden visual glitch with enchanted hand-held items was caused by having a hamster on my shoulder. With shaders on, there's a glitch with torches in hand as well, I presume because they're light-emitting?

When I realized it was Animania, I searched and found this existing issue.

It doesn't have anything to do with hamster balls, as noted in the Oct 3 comment.

I'm playing on 1.12.2.
